require('./utils').release().catch(err => {
  console.error(err)
})